test/integration/output_test.py

```diff
@@ -43,7 +46,7 @@ def fast_out_annual(pudl_engine, pudl_datastore_fixture):
         pudl_engine,
         ds=pudl_datastore_fixture,
         freq="AS",
-        fill_fuel_cost=True,
+        fill_fuel_cost=FILL_FUEL_COST,
```

Comment:
Had to add this reference, since it was not relying on the dynamically set value previously.